### PR TITLE
dif hackathon blog

### PR DESCRIPTION
**Pull request by EbonyLouis**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Direct link to preview: https://deploy-preview-1141--tbd-website-developer.netlify.app/blog/2024-01-09-dif-hackathon-winners

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206261662701725

### CODE REVIEW

**Comment by EbonyLouis** [comment]
my feelings wont be hurt if we hate the banner LOL

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+## DIF 🤝 Web5 Hackathon
+
+The results are in! Out of **418 participants** we have our **top 3** projects! We were honored to sponor a Web5 challenge with DIF Foundation. DIF (Decentralized Identity Foundation) is an engineering-driven organization focused on developing the foundational elements necessary to establish an open and interoperable ecosystem for decentralized identity. The challenge given was to **build a decentralized web app utilizing Web5.js and Decentralized Web Nodes**.
+
+<!--truncate-->
+
+## Hackathon Winners
+
+## 1. Anonymous Door Unlocking with Anonymity Revocation
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:46]:
```suggestion
## 🥇 Anonymous Door Unlocking with Anonymity Revocation
```

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowFullScreen
+        className="absolute top-0 left-0 w-full h-full"
+      ></iframe>
+    </div>
+  </div>
+</div>
+
+### Try it Out:
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:67]:
```suggestion
**Try it Out**
```

removing these from the side navbar

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- [GitHub Repo](https://github.com/JidoTaikaSuru/dwn-vc-job-market-demo)
+
+To learn more about their project and their experience checkout their [project's showcase page](https://devpost.com/software/an-anonymous-door-unlocking-with-anonymity-revocation).
+
+## 2. Decentralinked
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:73]:
```suggestion
## 🥈Decentralinked
```

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowFullScreen
+        className="absolute top-0 left-0 w-full h-full"
+      ></iframe>
+    </div>
+  </div>
+</div>
+
+### Try it Out:
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:94]:
```suggestion
**Try it Out**
```

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+- [Decentralinked](https://dorakemon-wallet.web.app/)
+- [GitHub Repos](https://github.com/dorakemon/did-hack-2023_winter)
+
+To learn more about their project and their experience checkout their [project's showcase page](https://devpost.com/software/decentralinked).
+
+## 3. HealthX Protocol
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:101]:
```suggestion
## 🥉HealthX Protocol
```

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowFullScreen
+        className="absolute top-0 left-0 w-full h-full"
+      ></iframe>
+    </div>
+  </div>
+</div>
+
+### Try it Out:
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:122]:
```suggestion
**Try it Out**
```

File: site/blog/2024-01-09-dif-hackathon-winners.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1. Anonymous Door Unlocking with Anonymity Revocation
+
+Anonymous Door Unlocking with Anonymity Revocation, was developed by a team from Sako laboratory of Waseda University in Tokyo, comprising of [Dorakemon Watanabe](https://github.com/dorakemon), [Kohei Morita](https://devpost.com/mrtkhgm), [Md Zaki Muzahid](https://devpost.com/md_zaki), [0xvon Tsutsumi](https://devpost.com/0xvon), and [Haruto Kimura](https://devpost.com/harutokimura0608). It's a solution designed for secure and private access control in lab environments.
+
+<div className="flex justify-center">
+  <div className="w-full sm:w-auto sm:max-w-[560px]">
+    <div className="overflow-hidden relative" style={{ paddingTop: '56.25%' }}>
+      <iframe
+        width="560"
```
Comment (angiejones) [inline site/blog/2024-01-09-dif-hackathon-winners.md:54]: we use the `aspect-video` class to ensure the video is responsive across device sizes. Here's an example of another video on our site:

```html
<iframe class="aspect-video" src="https://www.youtube.com/embed/j7Tm5L2P7hc?si=ajBdhkuqgJJ2lF5J" title="Verifiable Credentials" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

**Review by angiejones — CHANGES_REQUESTED** [review]

**Review by angiejones — APPROVED** [review]

**Pull request merged**